packages/paste-codemods/tools/mappings.json

```diff
@@ -0,0 +1,154 @@
+{
```

Comment:
Is this mappings file generated?

Reply:
Oh yep. Probably by the create-package-mappings.js file.

Reply:
Yup!